docs/user-guide/kubectl/kubectl_run.md

```diff
@@ -135,7 +135,7 @@ $ kubectl run nginx --image=nginx --command -- <cmd> <arg1> ... <argN>
 
 * [kubectl](kubectl.md)	 - kubectl controls the Kubernetes cluster manager
 
-###### Auto generated by spf13/cobra at 2015-09-25 23:39:47.894967733 +0000 UTC
+###### Auto generated by spf13/cobra at 2015-10-21 20:47:13.475967912 +0000 UTC
```

Comment:
Fixed it.
We need to run versionize-docs on pkg/kubectl/cmd as well, so that the generated kubectl docs dont contain HEAD links.